packages/kcms/src/match/routing.ts

```diff
@@ -70,7 +72,7 @@ export const GetMatchIdRoute = createRoute({
           schema: GetMatchIdResponseSchema,
         },
       },
-      description: 'Retrieve all matches by match type',
+      description: 'Retrieve all matches by matchId',
```

Comment:
Match1つが返ってくるので
```suggestion
      description: 'Retrieve match by matchID',
```
です

Reply:
`Retrieve a match by matchID`が適切でしょうか。